.github/workflows/release-ionic.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       uses: ./.github/workflows/actions/download-archive
       with:
         name: ionic-docs
-        path: ./docs
+        path: .packages/docs
```

Comment:
Shouldn't this be:
```
path: ./packages/docs
```

Reply:
oops https://github.com/ionic-team/ionic-framework/pull/29256/commits/9c6927d429ffd171286ed532c1fdf5487a2ddc31